Autopilot/PathManager/Inc/trackFinder.hpp

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+
+/*
+    The goal of this function is to return the current track of the plane, given its velocities in the latitude and longitude direction.
+    Given the speeds of the aircraft in the latitude and longitude directions in m/s, this function will output the track in degrees.
+*/
```

Comment:
It may be nitty gritty, but we should enforce max documentation. See how other prototypes are documented and follow that pattern. You'l need an @param[in] for each of your args to describe what it is, and an @return to describe exactly what your function returns.

Also the 2 sentences in your doc string are exactly the same. You only need 1.